README.md

```diff
@@ -130,11 +132,45 @@ if err != nil {
 If you have chosen Verify Condition on the Yoti Dashboard with the age condition of "Over 18", you can retrieve the user information with the generic .GetAttribute method, which requires the result to be cast to the original type:
 
 ```Go
-userProfile.GetAttribute("age_over:18").Value.(string)
+userProfile.GetAttribute("age_over:18").Value().(string)
 ```
 
 GetAttribute returns an interface, the value can be acquired through a type assertion.
 
+### Anchors, Sources and Verifiers
+
+An `Anchor` represents how a given Attribute has been _sourced_ or _verified_.  These values are created and signed whenever a Profile Attribute is created, or verified with an external party.
+
+For example, an attribute value that was _sourced_ from a Passport might have the following values:
+
+`Anchor` property | Example value
+-----|------
+type | SOURCE
+value | PASSPORT
+subType | OCR
+signedTimestamp | 2017-10-31, 19:45:59.123789
+
+Similarly, an attribute _verified_ against the data held by an external party will have an `Anchor` of type _VERIFIER_, naming the party that verified it.
+
+From each attribute can retrieve the `Anchors`, and subsets `Sources` and `Verifiers` (all as `[]*anchor.Anchor`) as follows:
+
+```Go
+givenNamesAnchors := userProfile.GivenNames().Anchors()
+givenNamesSources := userProfile.GivenNames().Sources()
+givenNamesVerifiers := userProfile.GivenNames().Verifiers()
+```
+
+You can also retrieve further properties from these respective anchors in the following way:
+
+```Go
+givenNamesFirstAnchor := userProfile.GivenNames().Anchors()[0]
+
+anchorType := givenNamesFirstAnchor.Type
+signedTimestamp := givenNamesFirstAnchor.SignedTimestamp().Timestamp
```

Comment:
Is Timestamp a DateTime Object?
If so, it will be good to add it as a comments.

Like this one -> // object, the actual timestamp as a *time.Time can be called with

Reply:
Yes it is a *time.Time. Instead of a comment, I'm going to specify the type explicitly on the LHS. Even though the compiler gives you a warning that the type can be inferred, and it's unnecessary, I think you're right in it's worthwhile to show the users what type to expect when we're displaying these in the README.